packages/adminrouter/extra/src/test-harness/tests/test_cache.py

```diff
@@ -948,45 +948,24 @@ def test_if_backend_requests_have_useragent_set_correctly(
 
 
 class TestCacheMarathon:
-
-    def test_ip_per_task_app_with_user_networking_and_portmappings(
-            self, nginx_class, mocker, valid_user_header):
-        app = self._scheduler_alwaysthere_app()
-        app['ipAddress'] = {'networkName': 'samplenet'}
-        app['tasks'][0]['ipAddresses'][0]['ipAddress'] = '127.0.0.2'
-        app['container']['docker']['network'] = "USER"
-        app['container']['docker']['portMappings'][0]['containerPort'] = '80'
-
-        ar = nginx_class()
-
-        mocker.send_command(endpoint_id='http://127.0.0.1:8080',
-                            func_name='set_apps_response',
-                            aux_data={"apps": [app]})
-
-        url = ar.make_url_from_path('/service/scheduler-alwaysthere/foo/bar/')
-        with GuardedSubprocess(ar):
-            # Trigger cache update by issuing request:
-            resp = requests.get(url,
-                                allow_redirects=False,
-                                headers=valid_user_header)
-
-            assert resp.status_code == 200
-            req_data = resp.json()
-            assert req_data['endpoint_id'] == 'http://127.0.0.2:80'
-
-    def test_ip_per_task_app_with_user_networking_and_portdefinitions(
-            self, nginx_class, mocker, valid_user_header):
+    @pytest.mark.parametrize('host_port', ['12345', '0', None])
+    def test_app_with_container_networking_and_fixed_container_port(
```

Comment:
Should we have a test for multiple `container` networks? (See the last point of [Marathon Port Mappings](https://mesosphere.github.io/marathon/docs/networking.html#port-mappings))

Reply:
We have got no request for supporting that case, so I think that would be out of the scope of this PR. We can add it as a feature work, if needed.